index.js

```diff
@@ -81,6 +82,49 @@ rbush3d.prototype = {
         return false;
     },
 
+    raycastInv: function (ox, oy, oz, idx, idy, idz, length) {
+        length = length || Infinity;
+        var node = this.data,
+            toBBox = this.toBBox,
+            result,
+            dist = length;
+
+        if (idx === Infinity && idy === Infinity && idz === Infinity) return undefined;
```

Comment:
Return a different value instead of undefined.  Overloading this can cause optimization problems.